### PR TITLE
DS 2.0 alpha release(En)

### PR DESCRIPTION
**Pull request by debrachena**
Hello community, good news! After nearly 10 months of joint efforts by more than 100 community contributors, we are happy to announce the release of Apache DolphinScheduler 2.0 alpha.

### CODE REVIEW

**Comment by debrachena** [comment]
Hello community, good news! After nearly 10 months of joint efforts by more than 100 community contributors, we are happy to announce the release of Apache DolphinScheduler 2.0 alpha.

**Comment by lenboo** [comment]
site-config also need to be changed.

File: site_config/blog.js [hunk 1]
```diff
@@ -3,7 +3,19 @@ export default {
...
-          {
+        
+      {
+        title: 'DS-2.0-alpha-release(En)',
+        author: 'Debbra Chen',
+        dateStr: '2021-10-29',
+        desc: 'DolphinScheduler 2.0 alpha release',
+        link: '/en-us/blog/DS-2.0-alpha-release(En).html',
+      },  
+
```
Comment (lenboo) [inline site_config/blog.js:14]: remove the blank to keep unified format

**Review by lenboo — APPROVED** [review]
+1

**Pull request merged**